packages/cdc-react/src/components/Pill/Pill.test.tsx

```diff
@@ -0,0 +1,94 @@
+import { fireEvent, render, screen } from "@testing-library/react";
+import { Pill } from "./Pill";
+
+describe("Pill Component", () => {
+  it("should render text", () => {
+    render(<Pill label="hello" />);
+    expect(screen.getByText("hello")).toBeInTheDocument();
+  });
+
+  it("should truncate long text", () => {
+    render(<Pill label="This is a long pull that should be truncated" />);
+    expect(
+      screen.getByTitle("This is a long pull that should be truncated")
+    ).toBeInTheDocument();
+  });
```

Comment:
This test seems to only be testing if the full text is being displayed, not if it is truncated. To test if it is truncated, try adjusting the text to the truncated version and see if it is being displayed.

Reply:
I think that checking for the class to be present is a good idea.

Reply:
Done